src/mrack/providers/aws.py

```diff
@@ -226,6 +231,12 @@ async def create_server(self, req):
         specs = deepcopy(req)  # work with own copy, do not modify the input
 
         del_vol = specs.get("delete_volume_on_termination", True)
+
+        # creating name for instance (visible in aws ec2 WebUI)
+        taglist = [{"Key": "name", "Value": req.get("name")}]
+        for key in self.instance_tags:
+            taglist.append({"Key": key, "Value": self.instance_tags[key]})
```

Comment:
Just style, i think this way is more readable and better  for looping dicts k/v
```suggestion
        for key, value in self.instance_tags.items():
            taglist.append({"Key": key, "Value": value)
```

Reply:
No prob i can update